WizardEngine/Player.cpp

```diff
@@ -45,12 +45,23 @@ Player::Player(Camera* a_Camera, ID3D11Device* device, ID3D11DeviceContext* cont
 
 	entityOneSpeed = 3.5;
 	wallRiseSpeed = 1.f;
+
+	playerHeight = 2.5f;
```

Comment:
The player/camera height should be stored somewhere so that vertical offset of all models in the scene can be changed easily.